Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1021,13 +1043,36 @@ std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElement
 
 std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElementBoneRotation(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
 {
+    if (boneId > BONE_RIGHTFOOT)
+    {
+        std::string error = "Invalid boneId: " + std::to_string(boneId);
+        m_pScriptDebugging->LogError(luaVM, error.c_str());
```

Comment:
You're using the new Lua parser. That means you have to throw exceptions if you want to print an error. See how it's implemented in other functions.

Reply:
use `throw LuaFunctionParser` with `false` as second parameter to throw an error

Reply:
Fixed, @TracerDS something like this doesn't even exist, no idea where did you get it from

Reply:
> Fixed, @TracerDS something like this doesn't even exist, no idea where did you get it from

sorry, I meant LuaFunctionError.